src/parser/jobs/rdm/Gauge.js

```diff
@@ -301,6 +394,51 @@ export default class Gauge extends Module {
 					<Trans id="rdm.gauge.suggestions.black-mana-lost-why">You lost {this._blackManaLostToImbalance} Black Mana due to overage of White Mana</Trans>
 				</Fragment>,
 			}))
+
+			this.suggestions.add(new TieredSuggestion({
+				icon: ACTIONS.VERHOLY.icon,
+				content: <Trans id="rdm.gauge.suggestions.wasted-verholy-content">
+					Do not use <ActionLink {...ACTIONS.VERFLARE}/> when you should use <ActionLink {...ACTIONS.VERHOLY}/>
```

Comment:
Minor (and not a blocker) but you might want to elaborate a bit in the content sections for all 3 of the new suggestions. Maybe something more like
```
Avoid using <ActionLink {...ACTIONS.VERFLARE}/> over <ActionLink {...ACTIONS.VERHOLY}/> when [reasons].
```
where `reasons` is all the valid reasons for flagging a cast as bad (procs, mana imbalance, what have you).